### PR TITLE
Update deployment.yaml - fix container

### PR DESCRIPTION
**Pull request by gandazgul**

### CODE REVIEW

File: deployments/kubernetes/chart/forecastle/templates/deployment.yaml [hunk 1]
```diff
@@ -31,25 +31,21 @@ spec:
         {{- toYaml . | nindent 8 }}
       {{- end }}
       containers:
-      - securityContext: {{- toYaml .Values.forecastle.container.securityContext | nindent 12 }}
+      - name: {{ template "forecastle.name" . }}
+        securityContext: {{- toYaml .Values.forecastle.container.securityContext | nindent 12 }}
```
Comment (MuneebAijaz) [inline deployments/kubernetes/chart/forecastle/templates/deployment.yaml:35]:
```suggestion
        securityContext: {{- toYaml .Values.forecastle.container.securityContext | nindent 10 }}
```

File: deployments/kubernetes/chart/forecastle/templates/deployment.yaml [hunk 2]
```diff
@@ -77,6 +73,10 @@ spec:
 {{ toYaml .Values.forecastle.openshiftOauthProxy.resources | indent 10 }}
       {{- end }}
       {{- end }}
+      {{- if .Values.forecastle.deployment.securityContext }}
+      securityContext:
+        {{- toYaml . | nindent 10 }}
```
Comment (MuneebAijaz) [inline deployments/kubernetes/chart/forecastle/templates/deployment.yaml:78]:
```suggestion
        {{- toYaml .Values.forecastle.deployment.securityContext | nindent 8 }}
```

**Review by MuneebAijaz — CHANGES_REQUESTED** [review]

**Comment by MuneebAijaz** [comment]
hi @gandazgul ive requested some changes, i fear the mistake comes from the old versions of the chart. Please do run helm template to be sure about indentation and templating. thanks

**Comment by bnallapeta** [comment]
PR is stale. Closing it.

**Comment by gandazgul** [comment]
Sorry I completely lost track of this I've been using my branch which just works and forgot to check back here. Is the main chart now fixed? At the time this seemed critical the chart was not working at all.

**Pull request closed without merging**